packages/neo-one-cli/src/cmd/compile.ts

```diff
@@ -8,15 +8,32 @@ export const describe = 'Compiles a project and outputs code to a local director
 export const builder = (yargsBuilder: typeof yargs) =>
   yargsBuilder
     .string('outDir')
-    .describe('outDir', 'output code directory')
+    .describe('outDir', 'Output code directory')
     .string('path')
-    .describe('path', 'contract directory');
+    .describe('path', 'Contract directory')
+    .boolean('json')
+    .describe('json', 'Output the contract with the JSON format')
+    .boolean('avm')
+    .describe('avm', 'Output the contract with the AVM format')
+    .boolean('abi')
+    .describe('abi', 'Output ABI alongside other files')
+    .default('abi', true)
+    .boolean('debug')
+    .describe('debug', 'Output additional debug information')
+    .default('debug', false);
```

Comment:
I know this seems irrelevant but could you add periods at the end of each description? We do that everywhere else for CLI descriptions

Reply:
Just so it's consistent with all other CLI descriptions